crates/matrix-sdk/src/http_client/native.rs

```diff
@@ -139,10 +140,12 @@ impl HttpSettings {
             reqwest::Client::builder().user_agent(user_agent).timeout(self.timeout);
 
         if self.disable_ssl_verification {
+            warn!("SSL verification disabled!");
```

Comment:
There's no instrument attribute on this method, so it might be a bit hard to know why it's telling us this.
```suggestion
            warn!("SSL verification disabled in the HTTP client!");
```